### DISCUSSION TITLE
Ideas for Endless Sky

### DISCUSSION THREAD

**Discussion by lumbar527** [endless-sky/endless-sky#10159] [Ideas]
I've been rereading the Thrawn Trilogy and have been thinking of few things I would like in Endless Sky. I'm not sure if this is the right way to present them, but here they are:

1. Ion weapons can disable ships. I don't mean what there already is, where ships are made inactive until they can restore energy, but that one you ion a ship enough (perhaps past a combination of `"ion resistance threshold"` attributes on reactors) you can board it.
2. Reasons for looting a ship rather than capturing it. This also ties into the various issues about capturing ships, namely the ones about locking equipment, but I would also have some interest in simple discouragement. Perhaps many shipyards won't want illegally obtained ships, or there's a chance for them to self destruct or something.
3. Gravity wells, like that actually pull ships in.
4. Hand-to-hand weapons that can be used up.
5. Custom mounts. They could specify if they are a gun port or turret mount but they would require a certain attribute. Application: Ships like the Emerald Sword have a weapon that can only go in one spot. Currently, this is less of a problem as the Emerald Sword only has one gun port, but you could still swap out the Dragonflame Cannon for an energy blaster.
Potentially more if I think of any.

**Comment by Zitchas** [comment]
Some good ideas.

You should go visit the center of the galaxy some time... The view is amazing...

**Reply by lumbar527** [reply]
Yeah, Sagittarius A* is cool.
In terms of gravity wells, I was referring to artificial ones. I guess my post isn't very clear on that.

**Comment by lumbar527** [comment]
5. Custom mounts. They could specify if they are a gun port or turret mount but they would require a certain attribute. Application: Ships like the Emerald Sword have a weapon that can only go in one spot. Currently, this is less of a problem as the Emerald Sword only has one gun port, but you could still swap out the Dragonflame Cannon for an energy blaster.
